lucene/sandbox/src/test/org/apache/lucene/sandbox/search/TestMultiRangeQueries.java

```diff
@@ -761,4 +763,90 @@ public void testEqualsAndHashCode() {
       assertNotEquals(query1.hashCode(), query3.hashCode());
     }
   }
+
+  private void addRandomDocs(RandomIndexWriter w) throws IOException {
+    Random random = random();
+    for (int i = 0; i < random.nextInt(100, 500); i++) {
```

Comment:
Let's not re-compute `nextInt` on every iteration.

```suggestion
    for (int i = 0, end = random.nextInt(100, 500); i < end; i++) {
```

Reply:
done